### PR TITLE
Fix start session logic on desktop lifecycle

### PR DESCRIPTION
**Pull request by aleksandr-dorofeev**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* ~[ ] Did you add unit tests if this modifies the Windows code?~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

The wrong variable was used to pass a value.
## Related PRs or issues

[AB#93380](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/93380)

### CODE REVIEW

File: SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelperDesktop.cs [hunk 1]
```diff
@@ -80,6 +79,7 @@ static ApplicationLifecycleHelperDesktop()
                     .GetField("Minimized")
                     .GetRawConstantValue();
             }
+            _suspended = false;
```
[the remark this answers is not in the document]
Comment (MatkovIvan) [reply SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelperDesktop.cs:84]: It doesn't explain why we need to do it here. Unresolved

**Review by AnatolyPristensky — APPROVED** [review]

File: SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelperDesktop.cs [hunk 2]
```diff
@@ -30,7 +30,6 @@ public class ApplicationLifecycleHelperDesktop : ApplicationLifecycleHelper
         // Need to ensure delegate is not collected while we're using it,
         // storing it in a class field is simplest way to do this.
         private static WinEventDelegate hookDelegate = new WinEventDelegate(WinEventHook);
-        private static bool suspended = false;
         private static bool started = false;
```
Comment (MatkovIvan) [inline SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelperDesktop.cs:33]: It should be reused from base class too

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelper.cs [hunk 3]
```diff
@@ -13,11 +13,20 @@ public abstract class ApplicationLifecycleHelper : IApplicationLifecycleHelper
...
         /// Indicates whether the application is currently in a suspended state. 
         /// </summary>
         public bool IsSuspended => _suspended;
 
+        /// <summary>
+        /// Indicates whether the application displays the active window.
+        /// </summary>
+        public bool HasShownWindow => _started;
+
+
```
Comment (MatkovIvan) [inline SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelper.cs:29]:
```suggestion
```

File: Tests/Microsoft.AppCenter.NET/ApplicationLifecycleHelper.cs [hunk 4]
```diff
@@ -16,6 +16,8 @@ public static IApplicationLifecycleHelper Instance
         }
         public bool IsSuspended { get; private set; }
 
+        public bool HasShownWindow { get; private set; }
```
Comment (MatkovIvan) [inline Tests/Microsoft.AppCenter.NET/ApplicationLifecycleHelper.cs:19]: It looks you're not updating this flag in this implementation

**Review by MatkovIvan — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**